packages/react/src/ReactCache.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import ReactCurrentCache from './ReactCurrentCache';
+
+const UNTERMINATED = 0;
+const TERMINATED = 1;
+const ERRORED = 2;
+
+type UnterminatedCacheNode<T> = {
+  s: 0,
```

Comment:
can we have some brief comments about what s v o p are?

Reply:
I assume you know and are asking just asking for inline comments, but just in case:

- v: value, either the cached result or an error, depending on `s`
- s: status, represents whether the cached computation returned a value or threw an error
- o: object cache, a WeakMap where non-primitive arguments are stored
- p: primitive cache, a regular Map where non-primitive arguments are stored.

Reply:
I low key thing we should rename the Fiber fields to single letters. Fight me.

Reply:
Or use Closure's advanced mode... somehow

Reply:
I wonder if it's really that infeasible now that DevTools is in the same repo.

Reply:
Once we get rid of the event system and clean up the DOM parts it'll be much easier.

Reply:
Why not use special identifiers like `.$child` and rename to letters at compile time. New letters on each release.

Reply:
Well my proposal was because the names are too long in development and don't really make much sense anyway. E.g. `memoizedState`